parserasgeo/features/cross_section.py

```diff
@@ -574,35 +574,29 @@ def alter_channel_n(self, scalar):
                     if old_n[0] == new_n[0]:
                         self.mannings_n.values.pop(ind)
                         self.mannings_n.values.insert(ind, new_n)
-                        break
-                
+                        break                
             self.channel_n = new_channel_n
-        else:
-            raise ChannelNError('The channel is undefined. Run define_channel_n before using alter_channel_n')
-
-    def bank_station_change(self, constant):
+            
+    def alter_nonchannel_n(self, scalar):
         """
-        Move the bank stations by constant [ft] if the user deems the original model to not be accurate
-        If constant > 0, the stations move outward
-        If cosntant < 0, the stations move inward
+        Alters the non-channel n-values by a scaling factor.
         
-        :param constant: constant by which the stations are moved
-        :returns: None
-        :raises BankStationError: riases error if moving the bank statiosn crates impossible situations (e.g. left bank > right bank)
+        :param scalar: a number by which the channel n values are scaled
+        :raises ChannelNError: raises error if channel_n not defined
         """
         
-        self.bank_sta.left -= constant
-        self.bank_sta.right += constant
-        
-        if self.bank_sta.left >= self.bank_sta.right:
-            raise BankStationError('left bank >= right bank for ({}, {}, {})'.format(self.header.station_id, self.river, self.reach))
-        
-        if self.bank_sta.left < 0:
-            raise BankStationError('left bank < 0 for ({}, {}, {})'.format(self.header.station_id, self.river, self.reach))
-        
-        if self.bank_sta.right > self.sta_elev.points[-1][0]:
-            raise BankStationError('right bank > last station for ({}, {}, {})'.format(self.header.station_id, self.river, self.reach))
-
+        if self.channel_n is None:
+            raise ChannelNError('The channel is undefined. Run define_channel_n before using alter_nonchannel_n')
+        else:
```

Comment:
Please delete this else in the interest of flatter code, and shift everything from new line 591 to 599 left 4 spaces. Same functionality but easier to read.